src/lib/autoslow.ts

```diff
@@ -78,7 +84,18 @@ export class AutoSlowManager {
       return slowMode;
     } else {
       const optimal = (Math.max(slowMode, 1) * currentBalance) / targetBalance;
-      return Math.min(Math.max(optimal, this.minSlow), this.maxSlow);
+      const upwardsChange = Math.max(
+        this.minAbsoluteChange,
+        slowMode * this.minChangeRate
+      );
+      const downwardsChange = Math.max(
+        this.minAbsoluteChange,
+        slowMode / (1 + this.minChangeRate)
```

Comment:
Since we're back to using multiplier, it makes sense that this is the multiplier itself, the suggested 2 would then be slowmode/2 instead of /3, which would be wrong.
```suggestion
        slowMode / this.minChangeRate
```